src/CattyTests/Project/ProjectMigratorTest.swift

```diff
@@ -0,0 +1,144 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+class ProjectMigratorTest: XCTestCase {
+
+    var newFolderName = String()
+    var header: Header!
+    var project: Project!
+
+    override func setUp() {
+        super.setUp()
+        newFolderName = kLocalizedScene
+
+        self.header = Header()
+        header.catrobatLanguageVersion = "0.992"
+
+        project = Project()
+        project.header = header
+    }
+
+    func testMigrateToScene() {
+        var automaticScreenShortAtPath = "\(String(describing: project.projectPath()))automatic_screenshot.png"
+        var imageDirectoryAtPath = "\(String(describing: project.projectPath()))images"
+        var soundDirectoryAtPath = "\(String(describing: project.projectPath()))sounds"
+
+        let fileManager = CBFileManagerMock(filePath: [automaticScreenShortAtPath], directoryPath: [imageDirectoryAtPath, soundDirectoryAtPath])
+
+        let migrate = ProjectMigrator(fileManager: fileManager)
+
+        do {
+            try migrate.migrateToScene(project: project)
+        } catch {
+            XCTFail("Some error occured")
+        }
+
+        automaticScreenShortAtPath = "\(String(describing: project.projectPath()) + newFolderName) 1/automatic_screenshot.png"
+        imageDirectoryAtPath = "\(String(describing: project.projectPath()) + newFolderName) 1/images"
+        soundDirectoryAtPath = "\(String(describing: project.projectPath() + newFolderName)) 1/sounds"
+
+        XCTAssertTrue(fileManager.fileExists(automaticScreenShortAtPath))
```

Comment:
Please also verify that the other, old files do not exist anymore, since the migrator should move those files and not create new files/directories.